examples/carousel/esp-idf/s2-kaluga-kit/main/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+# Copyright © SixtyFPS GmbH <info@slint.dev>
+# SPDX-License-Identifier: MIT
+
+idf_component_register(
+    SRCS "../../main.cpp"
+    INCLUDE_DIRS "."
+    REQUIRES slint
+)
+
+slint_target_sources(${COMPONENT_LIB} ../../../ui/carousel_demo.slint)
+target_link_options(${COMPONENT_LIB} PUBLIC -Wl,--allow-multiple-definition)
+set_target_properties(${COMPONENT_LIB} PROPERTIES SLINT_EMBED_RESOURCES "embed-for-software-renderer")
```

Comment:
Is it possible to make this in our slint component?

Reply:
Yes. I'll fix that - good idea.